src/ResourceManager/DeviceProvisioningServices/Commands.DeviceProvisioningServices/packages.config

```diff
@@ -0,0 +1,5 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="Microsoft.Azure.Management.DeviceProvisioningServices" version="0.9.0-preview" targetFramework="net452" />
+  <package id="Microsoft.Azure.Management.IotHub" version="1.1.3" targetFramework="net452" />
```

Comment:
Unfortunately, you can't reference Microsoft.Azure.Management.IotHub SDK from withing your project because this will lead to the assemblies clash. You have 2 options here:
1. Move the functionality you need into a common project and reference the project form the IotHub.and and DeviceProvisioningServices projects;
2. Regenerate code from the swagger spec but under a different namespace and use the code in your project.

Reply:
@vladimir-shcherbakov  Can you describe the "assemblies clash" mentioned in this feedback? These are both Nuget packages from the azure-sdk-for-net repo using "vanilla" AutoRest-generated clients from ARM calls (https://www.nuget.org/packages/Microsoft.Azure.Management.IotHub/ and https://www.nuget.org/packages/Microsoft.Azure.Management.DeviceProvisioningServices/0.9.0-preview). The assemblies should therefore already be in separate namespaces and able to be consumed independently or together within other assemblies.